.github/CODEOWNERS

```
@@ -57,24 +57,25 @@
 /compiler/src/iree/compiler/ @benvanik
 /compiler/src/iree/compiler/Codegen/ @MaheshRavishankar
 /compiler/src/iree/compiler/Codegen/Common @hanhanW @Max191
-/compiler/src/iree/compiler/Codegen/Common/GPU @antiagainst @qedawkins @Groverkss @kuhar
+/compiler/src/iree/compiler/Codegen/Common/GPU @qedawkins @Groverkss @kuhar
 /compiler/src/iree/compiler/Codegen/Dialect/Codegen @Max191
 /compiler/src/iree/compiler/Codegen/Dialect/CPU @hanhanW
-/compiler/src/iree/compiler/Codegen/Dialect/GPU @antiagainst @qedawkins @Max191
+/compiler/src/iree/compiler/Codegen/Dialect/GPU @qedawkins @Max191 @Groverkss @nirvedhmeshram @krzysz00
 /compiler/src/iree/compiler/Codegen/Dialect/VectorExt @Groverkss
 /compiler/src/iree/compiler/Codegen/ExternalInterfaces @hanhanW @Max191
-/compiler/src/iree/compiler/Codegen/LLVMCPU/ @hanhanW @MaheshRavishankar @pashu123
-/compiler/src/iree/compiler/Codegen/LLVMGPU/ @MaheshRavishankar @qedawkins @kuhar @Groverkss
-/compiler/src/iree/compiler/Codegen/SPIRV/ @antiagainst @MaheshRavishankar @kuhar
-/compiler/src/iree/compiler/ConstEval/ @hanhanW @stellaraccident
+/compiler/src/iree/compiler/Codegen/LLVMCPU/ @hanhanW
+/compiler/src/iree/compiler/Codegen/LLVMGPU/ @qedawkins @kuhar @Groverkss @Groverkss  @nirvedhmeshram  @krzysz00 @Max191
```

```suggestion
/compiler/src/iree/compiler/Codegen/LLVMGPU/ @qedawkins @kuhar @Groverkss @nirvedhmeshram  @krzysz00 @Max191
```

I think @Groverkss does not need to review this twice. :)

---

(and dropping the extra whitespace)